dftimewolf/lib/exporters/timesketch.py

```diff
@@ -84,14 +95,13 @@ def launch_exporter(
       A list of TimesketchExporter objects that can be join()ed from the caller.
     """
 
-    print('Using timesketch endpoint: {0:s}'.format(ts_endpoint))
-
-    timesketch_api = timesketch_utils.TimesketchApiClient(
-        ts_endpoint, ts_username, ts_password)
+    auth = (ts_endpoint, ts_username, ts_password)
```

Comment:
As above, this is weird, don't do this, just pass the args. Also, timesketch_endpoint etc.